packages/guess-webpack/.gitignore

```diff
@@ -0,0 +1,3 @@
+api/index.js
```

Comment:
Publishing `guess-api` as a separate package will simplify things a bit and remove the need for `.gitignore` on level individual files.